### PR TITLE
added notice for insufficient space error w/ volume resizing

### PR DESCRIPTION
**Pull request by ejschaefer**
Added tool tip to CDK init steps with information about insufficient space errors and remediation using resize.sh script

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ejschaefer** [comment]
Updated bootstrap.sh to skip homebrew installation is it already exists. Cloud9 Ubuntu and AmazonLinux AMI are pre-baked with homebrew installed and .profile/.bash_profile updated

fixes #32

File: workshop/static/assets/bootstrap.sh [hunk 1]
```diff
@@ -60,13 +64,18 @@ function install_utility_tools() {
...
-    test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
-    test -d /home/linuxbrew/.linuxbrew && eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
-    test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.bash_profile
-    echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.profile
+
+    if [[ $(command -v brew) == "" ]]; then
+        _logger "[+] Installing homebrew..."
+        echo | sh -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+        _logger "[+] Adding homebrew in PATH"
+        test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
```
Comment (ferdingler) [inline workshop/static/assets/bootstrap.sh:72]: Are the installation paths still `/home/linuxbrew` and `~/.linuxbrew` even though the download URL changed from Linuxbrew to Homebrew?

Comment (ejschaefer) [reply workshop/static/assets/bootstrap.sh:72]: Yes, the homebrew installer checks if the OS is Linux or MacOS. If the OS is Linux and the user has sudo access it installs to `/home/linuxbrew` or `~/.linuxbrew` if the user does not have sudo access. 

These lines will check for either install type so environment variables can be set 
``` 
  test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
  test -d /home/linuxbrew/.linuxbrew && eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
```

Comment (ferdingler) [reply workshop/static/assets/bootstrap.sh:72]: Ok, thanks. I will merge now.

**Pull request merged**